src/go/k8s/kuttl-v2-ci-test.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestSuite
+startKIND: true
+kindContainers:
+  - localhost/redpanda-operator:dev
+  - docker.redpanda.com/redpandadata/redpanda:v23.2.8
+  - docker.redpanda.com/redpandadata/console:v2.3.2
```

Comment:
NIT: I would remove this as this is already outdated

REF:
https://github.com/redpanda-data/helm-charts/blob/214b5cec548086ea6590b77bfe1b694c22f6ed9b/charts/console/Chart.yaml#L33
https://github.com/redpanda-data/helm-charts/blob/214b5cec548086ea6590b77bfe1b694c22f6ed9b/charts/redpanda/Chart.yaml#L30